common/workunit/workunit.cpp

```diff
@@ -6314,6 +6314,13 @@ void CLocalWorkUnit::loadPTree(IPropertyTree *ptree)
 {
     clearCached(false);
     p.setown(ptree);
+#ifdef _CONTAINERIZED
+    if (getDebugValueBool("forceRemoteRead", false))
+    {
+        WARNLOG("Remote reading via dafilesrv not supposed within containers - disabling");
```

Comment:
typo - supposed should say supported.

Not sure it's really worth warning though?

Reply:
ok, yes wasn't sure. I'll remove the warning.